src/__tests__/issues/737-org-authors.ts

```diff
@@ -29,13 +29,12 @@ test('issue 737: references with an organisational author', async () => {
   const bib = await new BibCodec().dump(ref)
   expect(bib).toBe(
     `@article{bib7,
+\tauthor = {{Malaria Genomic Epidemiology Network} and Rockett, KA and Clarke, GM and Fitzpatrick, K and Hubbart, C and Jeffreys, AE and Rowlands, K and Craik, R and Jallow, M and Conway, DJ},
 \tjournal = {Nature Genetics},
+\tyear = {2014},
+\tpages = {1197--1204},
 \ttitle = {Reappraisal of known malaria resistance loci in a large multicenter study},
 \tvolume = {46},
-\tauthor = {{Malaria Genomic Epidemiology Network} and Rockett, KA and Clarke, GM and Fitzpatrick, K and Hubbart, C and Jeffreys, AE and Rowlands, K and Craik, R and Jallow, M and Conway, DJ},
-\tpages = {1197--1204},
-\tdate = {2014},
-\tyear = {2014},
```

Comment:
This and the next comment are actually unrelated to this PR (I pushed branches 'out of order'). They are related to an update to Citation.js.

Nonetheless, well spotted. Yes this was intentional as Citation.js is now only outputting `year` in BibTeX presumably because of aforementioned duplication .